.travis.yml

```
@@ -2,7 +2,7 @@ language: python
 python:
   - "3.8"
 # command to install dependencies
-install: "pip install -r requirements.txt"
+install: "pipenv install"
```

You might try this to make Travis CI play more nicely with Pipenv:

```suggestion
install: "pip install $(pipenv lock --requirements)"
```